atomic_reactor/plugin.py

```diff
@@ -210,6 +214,9 @@ def __init__(self, dt, workflow, plugin_class_name, plugins_conf, *args, **kwarg
         self.workflow = workflow
         super(BuildPluginsRunner, self).__init__(plugin_class_name, plugins_conf, *args, **kwargs)
 
+    def on_plugin_failed(self):
+        self.workflow.build_has_failed = True
```

Comment:
Right. Now I finally get it.

So the variable will reflect two states:
1. `docker build` has failed
2. one of the plugins has failed

Wouldn't it make sense rather use two variables for that?